### ISSUE TITLE
Ctrl!+!X! not exact

### ISSUE THREAD

**Issue by chrvip** [Odin94/intellij-shortcuts-anki-deck#1]
in your deck is "**cut line**"
In fact in refrence card in idea，it says:

> Ctrl+X Cut current line or selected block to clipboard

**Comment by Odin94** [comment]
This is true. I consider Ctrl+X cutting the selected block a default text editing feature that works everywhere, much like Ctrl+C for copying the selected block. 

Ctrl+X cutting the current line if no text is selected however does not work in all text editors.

For that reason I did not include Ctrl+X cutting the selected block in the card.
